module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,18 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+std::deque<std::string> lengthSort(const std::forward_list<std::string>& list){
+    std::deque<std::string> queue;
+    for(const auto& el : list){
+        queue.push_back(el);
+    }
```

Comment:
We don't need this push.back loop. We can initialize deque like this:
std::deque<std::string> queue {list.begin(), list.end()};
BTW naming deque "queue" is a little confusing since there separate types of structure.